zadani/soft/index.html

```diff
@@ -1,96 +1,138 @@
+            <p>Island je domovem největšího evropského ledovce, vodopádu s nejmohutnějším průtokem a tisíců místních koní a divokých hus, které majestátně ve dvojicích křižují nebe a dokreslují tak neopakovatelnou atmosféru tohoto pozoruhodného kousku pevniny
+                kdesi v Atlantiku. K tomu mohutné skalní útvary, čnící z moře jako tiší strážci místních černých lávových pláží, stovky metrů vysoké útesy, které jsou domovem racků a papuchalků a především obrovské vodopády všeho druhu. Panují tu trochu
+                jiná měřítka, a to téměř ve všem.</p>
 
-        <p>Severní pobřeží je pak místy opravdová divočina, už tak mizivé osídlení ještě více prořídne a vy projíždíte místní fjordy po šotolinových cestách, aniž byste hodiny potkali jediné auto. Nad hlavou vám ční vysoké hory pokryté věčným sněhem, každým údolím se v kaskádách a vodopádech řítí dolů řeky, prostě divočina zamrznutá v čase. Dostane se vám to tu pod kůži a nutí Vás to přemýšlet, jak je ve skutečnosti člověk maličký a nicotný&hellip;</p>
+            <p>Pociťujete tu zvláštní volnost. Mimo tří větších měst, kde je soustředěna drtivá většina obyvatel, můžete projíždět krajinou stovky kilometrů a čas od času potkáte jen osamělou usedlost, obklopenou stády koní a ovcí. Na jižním pobřeží když
+                už narazíte na vesnici, jedná se o seskupení usedlostí, jejichž počet byste kolikrát spočítali na prstech jedné ruky a samozřejmě nesmí chybět benzínová pumpa. Právě čerpací stanice jsou zde centrem dění.</p>
 
+            <p>Severní pobřeží je pak místy opravdová divočina, už tak mizivé osídlení ještě více prořídne a vy projíždíte místní fjordy po šotolinových cestách, aniž byste hodiny potkali jediné auto. Nad hlavou vám ční vysoké hory pokryté věčným sněhem,
+                každým údolím se v kaskádách a vodopádech řítí dolů řeky, prostě divočina zamrznutá v čase. Dostane se vám to tu pod kůži a nutí Vás to přemýšlet, jak je ve skutečnosti člověk maličký a nicotný&hellip;</p>
 
-        <img src="../../images/island2.jpg" alt="Island">
-        <p class="popisek">
-            Photo &copy; <a href="https://flic.kr/p/yiAg2h">Moyan Brenn</a>
-        </p>
+        </section>
 
+        <div class="islandfoto">
+            <img src="../../images/island2.jpg" alt="Island">
+            <p class="popisek1">
+                Photo &copy; <a href="https://flic.kr/p/yiAg2h">Moyan Brenn</a>
+            </p>
+        </div>
 
         <!-- sekce Co musíte vidět -->
 
-        <h2>Co musíte vidět</h2>
+        <section class="turistika">
```

Comment:
U sekcí bych strukturu trochu upravila. Máme tu vlastně 2 druhy sekcí - sekce "levé" a sekce vpravo.
Takže určitě nepotřebujeme vytvářet třídu pro každou sekci zvlášť, když nejsou ničím jedinečné co se týká vzhledu. 
Třída má právě tu skvělou vlastnost, že je "recyklovatelná" a šetří nám psaní. Takže podle toho, kolik stylování potřebujeme bych udělala maximálně 3 třídy: `sekce` `sekce-leva`, `sekce-prava` (pojmenování je samozřejmě na tobě) 🙂